cpp/src/arrow/compute/kernels/scalar_temporal_test.cc

```diff
@@ -2159,13 +2162,17 @@ TEST_F(ScalarTemporalTest, StrftimeOtherLocale) {
 }
 
 TEST_F(ScalarTemporalTest, StrftimeInvalidLocale) {
+#ifdef EMSCRIPTEN
+  GTEST_SKIP() << "Emscripten doesn't build with multiple locales as default";
+#else
```

Comment:
We don't need `#else`.

```suggestion
#endif
```